lib/src/embedded/compiler.ts

```diff
@@ -7,8 +7,21 @@ import {EmbeddedProcess} from './compiler/process';
 import {PacketTransformer} from './compiler/packet-transformer';
 import {MessageTransformer} from './compiler/message-transformer';
 import {Observable, merge} from 'rxjs';
-import {filter, first, map} from 'rxjs/operators';
-import {Dispatcher} from './dispatcher';
+import {filter, first, map, take, takeUntil, tap} from 'rxjs/operators';
+
+type Request =
+  | InboundMessage.CompileRequest
+  | OutboundMessage.ImportRequest
+  | OutboundMessage.FileImportRequest
+  | OutboundMessage.CanonicalizeRequest
+  | OutboundMessage.FunctionCallRequest;
```

Comment:
Why is it useful to group inbound and outbound requests under the same type?

Reply:
It's not :) This was a mistake corrected in a newer commit.